tests/jose-jwe-enc

```diff
@@ -1,29 +1,30 @@
-#!/bin/bash -ex
+#!/bin/sh -ex
 
 WRAP=`jose alg -k wrap`
 ENCR=`jose alg -k encr`
 
 tmpdir=`mktemp -d 2>/dev/null || mktemp -d -t jose`
 
-function onexit() {
+onexit() {
     rm -rf $tmpdir
 }
 
-trap exit ERR
+# not needed since the script is executed with -e
```

Comment:
Right, and probably this comment (and the other ones like it) are not needed either :)
Just remove the comment and the commented trap.